buildbot-conf/cordova-repos.json

```diff
@@ -1,457 +1,285 @@
 {
     "mobile-chrome-apps": {
-        "codebase":  "chrome",
-        "codebases": {
-            "chrome": {
-                "repo":   "https://github.com/MobileChromeApps/mobile-chrome-apps.git",
-                "branch": "master"
-            }
+        "default_repository": "chrome",
+        "default_branch":     "master",
+        "repositories":       {
+            "chrome": "https://github.com/MobileChromeApps/mobile-chrome-apps.git"
         }
     },
 
     "cordova-medic": {
-        "codebase":  "github",
-        "codebases": {
-            "asf":  {
-                "repo":   "https://git-wip-us.apache.org/repos/asf/cordova-medic.git",
-                "branch": "master"
-            },
-            "github":  {
-                "repo":   "https://github.com/apache/cordova-medic.git",
-                "branch": "master"
-            }
+        "default_repository": "msot",
```

Comment:
The default repository is "msot" but "repositories" contain only "asf" and "github".
Is that intended?